docs/examples/git-repo/gitrepo-kind.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: GitRepository
+metadata:
+  name: podinfo
+  namespace: default
+spec:
+  interval: 1m
+  url: https://github.com/wangyuan249/podinfo
+  ref:
+    branch: master
```

Comment:
This, together with the `kustomization` object seems to be the component you want to deploy. So the better solution is using CUE to glue them together.

Reply:
Yes, this should be a CUE template with GitRepo and Kustomize included.